README.md

```diff
@@ -101,6 +103,23 @@ Remember you'll need to run `bundle install` and `yarn install` on an ongoing ba
      - If you run into problems with `solr_wrapper`, you can also start Solr with `rake figgy:development`
    - Access Figgy at http://localhost:3000/
 
+### Lando
+
+1. Uninstall Docker
+2. Install Lando DMG from [[https://github.com/lando/lando/releases]]
+3. `lando start`
+
+1. For test:
+   - `RAILS_ENV=test rake db:setup`
+   - In a separate terminal: `bundle exec rspec`
+   - Run jest tests: `yarn test`
+2. For development:
+   - ``export SECRET_KEY_BASE=`rake secret` ``
+   - `rake db:setup`
+   - In a separate terminal: `foreman start` (you can close the one launching solr)
```

Comment:
Procfile still includes solr wrapper. This will conflict with the lando port so it should probably be removed.

Reply:
It doesn't conflict with the lando port, but it isn't necessary so I'll remove it.

Reply:
Hm. Not sure why foreman failed for me then. I don't generally use it so there may have been a pre-existing or unrelated issue. anyway 👍